qiskit/aqua/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -223,14 +223,14 @@ def is_good_state(bitstr):
 
         # Construct GroverOperator circuit
         if grover_operator is not None:
-            self._grover_operator = grover_operator
+            self.grover_operator = grover_operator
         else:
             # wrap in method to hide the logic of handling deprecated arguments, can be simplified
             # once the deprecated arguments are removed
-            self._grover_operator = _construct_grover_operator(oracle, state_preparation,
-                                                               mct_mode)
+            self.grover_operator = _construct_grover_operator(oracle, state_preparation,
```

Comment:
This was changed from a private variable to public it seems. Would it be better to have a getter for access to the private variable where it can be documented too. Is it the intent that this is accessible this way in general or was it done just for the unit tests?

Reply:
You are right. Making a getter is a better idea. I fixed it.